src/PrestaShopBundle/Resources/config/services/bundle/form/error_message.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  _defaults:
+    public: true
```

Comment:
I think these services don't need to be public

Reply:
As far as I see every single yml file in PS has services as public, I don't remember the exact reason, but I know at least in modules sometimes things don't work if you don't make services public.

Reply:
We are starting to make private services to optimize the container, I think it is better to stop adding public service, il will ease the work in the future.

Also, you can create your new services with the FQCN now :)

Reply:
Alright that makes sense. Implemented FQCN.